src/components/templates/TalkShowStudio/useStage.ts

```diff
@@ -0,0 +1,194 @@
+import { useCallback, useEffect, useMemo } from "react";
+
+import { MAX_TALK_SHOW_STUDIO_HOSTS } from "settings";
+
+import { PlaceInTalkShowStudioVenue, User } from "types/User";
+
+import { updateTalkShowStudioExperience } from "api/profile";
+import { updateUserTalkShowStudioExperience } from "api/admin";
+
+import { WithId } from "utils/id";
+
+import { useUser } from "hooks/useUser";
+import { useVenueId } from "hooks/useVenueId";
+import { useRecentVenueUsers } from "hooks/users";
+
+export const useStage = () => {
```

Comment:
This hook presumably shouldn't be in `src/components/templates/TalkShowStudio/useStage.ts` ?